test/integration/LockingResourceStore.test.ts

```diff
@@ -41,8 +42,9 @@ describe('A LockingResourceStore', (): void => {
     );
 
     // Initialize store
-    const initializer = new RootContainerInitializer({ store: source, baseUrl: base });
-    await initializer.handleSafe();
+    const metadata = new RepresentationMetadata({ path: base }, TEXT_TURTLE);
+    metadata.add(RDF.type, PIM.terms.Storage);
+    await source.setRepresentation({ path: base }, new BasicRepresentation([], metadata));
```

Comment:
Why not have the actual initializer do this?

Reply:
Because setting up the initializer then also requires setting up the resources generator and the templates handlers. So this was less lines of code. No real other valid reason :D

Reply:
Fair enough.